Utilities/Thread.cpp

```diff
@@ -1761,7 +1761,11 @@ static LONG exception_filter(PEXCEPTION_POINTERS pExp) noexcept
 
 const bool s_exception_handler_set = []() -> bool
 {
-	if (!AddVectoredExceptionHandler(1, (PVECTORED_EXCEPTION_HANDLER)exception_handler))
+#ifdef USE_ASAN
+	if (!AddVectoredExceptionHandler(FALSE, (PVECTORED_EXCEPTION_HANDLER)exception_handler))
+#else
+	if (!AddVectoredExceptionHandler(0, (PVECTORED_EXCEPTION_HANDLER)exception_handler))
```

Comment:
why did you change it to 0 ?

Reply:
Sorry, a mistake. I fixed it.

Reply:
also FALSE is 0, not sure why you use it.

Reply:
I used `FALSE` because I copied it from here: https://developercommunity.visualstudio.com/t/asan-problems-when-manual-exception-handling/1242524#T-N1295191 .